pkg/kwokctl/runtime/binary/cluster.go

```diff
@@ -50,6 +51,10 @@ type Cluster struct {
 
 // NewCluster creates a new Runtime for binary
 func NewCluster(name, workdir string) (runtime.Runtime, error) {
+	if goruntime.GOOS != "linux" {
+		return nil, fmt.Errorf("binary: only supported in linux")
+	}
+
```

Comment:
Binary is works in non-Linux

Reply:
then how should we detect whether binary works?

Reply:
I was thinking that the default order for Linux settings is docker, nerdctl and binary. and for everything else just check docker and nerdctl.

Reply:
like the previous one, it needs some refactoring to set the config, how about hardcoded currently?

users can always specify one if they want to

Reply:
I don't think hard coding is appropriate.

Reply:
Set up a default for KwokctlConfiguration https://github.com/kubernetes-sigs/kwok/blob/122f36ce87128371d2b9bbcff43837d764b35e37/pkg/config/vars.go#L123

Reply:
Oops, sorry for not noticed the config is already existed inside flags, will fix this